examples/break-statement/break_statement.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/io;
+
+public function main() {
+
+    string[] names = ["Bob", "Jo", "Ann", "Tom"];
+    int i = 0;
+    while true {
+        io:println(names[i]);
+        // loop breaks when condition satisfied
```

Comment:
The comments should start with an upper case letter and end with a full stop. Please check other places as well.